scvi/inference/trainer.py

```diff
@@ -128,7 +128,14 @@ def compute_metrics(self):
                 self.model.train()
         self.compute_metrics_time += time.time() - begin
 
-    def train(self, n_epochs=20, lr=1e-3, eps=0.01, params=None):
+    def train(
+        self,
+        n_epochs=20,
+        lr=1e-3,
+        eps=0.01,
+        max_number_of_consecutive_nans=10,
```

Comment:
Can we just call this `man_nans`? And then explain it's consecutive in the docstring?